doc/changelog.md

```diff
@@ -17,7 +17,7 @@ Description
 - Added schemas and MessageHandler class for de/serialization of
   inference requests and response messages
 - Removed device from schemas, MessageHandler and tests
-
+- Add TorchWorker first implementation and mock inference app example
```

Comment:
I don't know if I've been getting it wrong every time, but I always do "newest change on top" in the changelog! Can you confirm which is correct and make sure I do the right thing?

Reply:
Newest on top!